commit 0bb9d51965f1920f303f8c9abd8133ce99d5818f
Author: Nicholas Marriott <nicholas.marriott@gmail.com>
Date:   Wed Feb 26 21:42:59 2014 +0000

    Don't write into buffer if no arguments, reported by Filipe Rosset.

diff --git a/cmd.c b/cmd.c
index 5e6b93a..a7a71a6 100644
--- a/cmd.c
+++ b/cmd.c
@@ -138,6 +138,9 @@ cmd_pack_argv(int argc, char **argv, char *buf, size_t len)
 	size_t	arglen;
 	int	i;
 
+	if (argc == 0)
+		return (0);
+
 	*buf = '\0';
 	for (i = 0; i < argc; i++) {
 		if (strlcpy(buf, argv[i], len) >= len)
